requirment.txt

```diff
@@ -1 +1 @@
-1-bs4 package
\ No newline at end of file
+we need to intall bs4 package to run this scrapper
```

Comment:
??? 

This is not the way requirements.txt works, the file name is incorrect and this line in here is not needed/not how it is to be used.

Reply:
@mdehsan873 this is still unresolved